evaluation/config.json

```diff
@@ -0,0 +1,28 @@
+{
+    "testdata_path": "input/qa.jsonl",
+    "results_dir": "results/experiment<TIMESTAMP>",
+    "passing_rate":3,
+    "requested_metrics": ["gpt_groundedness", "gpt_relevance", "gpt_coherence", "answer_length", "latency"],
+    "target_url": "https://app-backend-xm5gmlddfm3cc.azurewebsites.net/ask",
```

Comment:
This is not something we want to keep in our config, because the target URL might be different between redeployments. This should be set along the rest of the deployment-specific values in environment variables.

Reply:
In your README you should also provide instructions on how one can find this. Perhaps it's already set in a local azd env var for instance and you can retrieve it directly from there.

Reply:
What if we need to run the evaluation against the locally running app, we can't retrieve the localhost URL from anywhere. Does the user need to set the env var manually in this case?

Reply:
> This is not something we want to keep in our config, because the target URL might be different between redeployments. This should be set along the rest of the deployment-specific values in environment variables.

What if we need to run the evaluation against the locally running app, we can't retrieve the localhost URL from anywhere. Does the user need to set the env var manually in this case?

Reply:
> What if we need to run the evaluation against the locally running app, we can't retrieve the localhost URL from anywhere.

That's a perfectly valid scenario!

> Does the user need to set the env var manually in this case?

Absolutely! You can still use the config for providing defaults. IIRC, they set the URL to a local one upstream. We can do the same and allow for overriding that through the envvars!